starter_code/src/movies.js

```diff
@@ -1,20 +1,87 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
+let turnHoursToMinutes = movies.map(movie => {
+    //apply the following function "movie" to each element of my array
+    var x = movie.duration.charAt(0) * 60 + parseInt(movie.duration.slice(3, 5));
+    //parseInt changes strings into integers
+    //movie.duration = get the property duration of the object movie in my array
+    movie.duration = x //now the property duration is equal to the equation created
+    return movie; //movie is each element of my array, with all it's properties
+})
+
+console.log(turnHoursToMinutes);
 
 
 // Get the average of all rates with 2 decimals 
+let ratesAverage = movies.reduce((accumulator, movie) => {
+    return accumulator + parseInt(movie.rate);
+}, 0)
+
+let averageRate = ratesAverage / movies.length
+console.log(averageRate)
 
 
 // Get the average of Drama Movies
+var count = 0
+let dramaMovieRate = movies.reduce((accumulator, movie) => {
+    if (movie.genre.includes("Drama")) {
+        count++;
+
+        return accumulator + parseInt(movie.rate);
```

Comment:
Logic is good, but the same problem as on line 17

Reply:
done!